ReactCommon/hermes/inspector/tools/run_msggen

```diff
@@ -2,6 +2,9 @@
 
 set -e
 
+ROOT=$(pwd)
+pushd $ROOT >/dev/null || exit
```

Comment:
You don't need `|| exit` as you have a `set -e` on top

Reply:
Fixed